### ISSUE TITLE
Display local image files in Viewer, or otherwise support `getOption('viewer')` for images

### ISSUE THREAD

**Issue by jeroen** [posit-dev/positron#2888]
In RStudio we can do:

```r
library(magick)
frink <- image_read("https://jeroen.github.io/images/frink.png")
print(frink)
```

But this does not work positron. I think the problem is that the `getOption('viewer')` does not work with images, only html?

**Comment by jmcphers** [comment]
Should be resolved with #4151:

<img width="769" alt="image" src="https://github.com/user-attachments/assets/ea7b7919-ede5-41b8-9a02-1fa379ca0d5c">

**Comment by testlabauto** [comment]
### Verified Fixed
    Positron Version(s) : 2024.07.0-107
    OS Version          : OSX

### Test scenario(s) 

library(magick)
frink <- image_read("https://jeroen.github.io/images/frink.png")
print(frink)

Link(s) to TestRail test cases run or created: 
N/A
